File: Maths -- Geometry/Python3/Palindrome Number(1) .py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False

        num = 0
        check = x
        while(check!=0):
            digit = check % 10
            num = digit + num*10
            check//=10

        if(x==num):
            return True
        else:
            return False
         

File: Maths -- Geometry/C++/Palindrome Number (1) .cpp
class Solution {
public:
    bool isPalindrome(long long int x) {
        long long int ans = x;
        if (x < 0) {
            return false;
        }
        long long int num;
        while (ans != 0) {
            int rem = ans % 10;
            num = num * 10 + rem;
            ans /= 10;
        }
        if (num == x) {
            return true;
        } else {
            return false;
        }
    }
};